File: 10000_20000/10000_11000/10870.cpp
#include <iostream>
using namespace std;

int fibo(int num) {
	if(num == 0)
		return 0;
	else if (num == 1)
		return 1;
	else
		return fibo(num-1) + fibo(num-2);
}

int main() {
	int n;
	cin >> n;
	cout << fibo(n) << '\n';
	
	
	return 0;
}

File: Python/BOJ/10870.py
# 피보나치 수 5
import sys
input = sys.stdin.readline
num = int(input())
def fibo(n):
    fi = [1,1]
    if n == 0:
        return 0
    if n == 1 or n == 2:
        return 1
    for i in range(2,n):
        fi.append(fi[i-1] + fi[i-2])
    return fi[num-1]
print(fibo(num))